nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -554,7 +555,7 @@ impl DataStore {
             // - `hw_baseboard` with an "id" primary key and lookup columns
             //   "part_number" and "serial_number"
             // - `sw_caboose` with an "id" primary key and lookup columns
-            //   "board", "git_commit", "name", and "version"
+            //   "board", "git_commit", "name", "version, and sign"
```

Comment:
```suggestion
            //   "board", "git_commit", "name", "version", and "sign"
```

(sorry just noticed this typo)